ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -695,6 +699,221 @@ literals:
   - PRESENT
   - ABSENT
 
+# Added for TPKD day2
+- condensing_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- condensing_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- oil_heater_status:
+  - ON
+  - OFF
+- inhibit_start_time_sensor
+- load_limit_capacity
+- total_compressor_count
+- compressor_run_time_sensor
+- guidevane_percentage_command
+- oil_temperature_sensor
+- oil_pump_differential_pressure_sensor
+- exhaust_gas_compressor_temperature_sensor
+- bearing_temperature_sensor
+- winding_temperature_sensor
+- compressor_temperature_setpoint
+- compressor_temperature_sensor
+- load_compressor_percentage_setpoint
+- compressor_power_sensor
+- refrigerant_valve_percentage_sensor
+- run_time_sensor
+- standby_compressor_alarm:
+  - ACTIVE
+  - INACTIVE
+- high_compressor_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_compressor_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- compressor_current_alarm:
+  - ACTIVE
+  - INACTIVE
+- compressor_run_time_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_high_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_overflow_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_low_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- fan_speed_frequency_setpoint
+- compressor_speed_frequency_setpoint
+- start_mode:
+  - ON
+  - OFF
+- reset_alarm:
+  - ACTIVE
+  - INACTIVE
+- reset_command:
+  - ON
+  - OFF
+- chilled_water_isolation_valve_mode:
+  - AUTO
+  - MANUAL
+- pump_speed_frequency_setpoint
+- chilled_water_isolation_valve_percentage_setpoint
+- chilled_water_flowrate_alarm:
+  - ACTIVE
+  - INACTIVE
+- end_alarm:
+  - ACTIVE
+  - INACTIVE
+- start_alarm:
+  - ACTIVE
+  - INACTIVE
+- simulation_command:
+  - ON
+  - OFF
+- condensing_supply_water_isolation_simulation_command:
+  - ON
+  - OFF
+- stop_command:
+  - ON
+  - OFF
+- closed_command:
+  - ON
+  - OFF
+- closed_alarm:
+  - ACTIVE
+  - INACTIVE
+- open_alarm:
+  - ACTIVE
+  - INACTIVE
+- simulation_status:
+  - ON
+  - OFF
+- local_status:
+    - REMOTE
+    - LOCAL
+- compressor_open_percentage_command
+- high_pressure_sensor
+- low_pressure_sensor
+- high_temperature_sensor
+- low_temperature_sensor
+- compressor_current_sensor
+- high_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- high_temperature_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_temperature_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_water_differential_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- chilled_supply_water_low_temperature_alarm:
+  - ACTIVE
+  - INACTIVE
+- chilled_water_differential_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- condensing_water_flowrate_alarm:
+  - ACTIVE
+  - INACTIVE
+- current_alarm:
+  - ACTIVE
+  - INACTIVE
+
+- motor_energy_sensor
+- motor_frequency_sensor
+- heat_percentage_sensor
```

Comment:
what is the context for this?

Reply:
heat_percentage_sensor - Motor Heat Load (Motor heat loss, calculated by output mechanical power from the input electrical power)

Reply:
wouldnt this be in units of power then? something like heat_loss_thermal_power_sensor?

Reply:
The unit is percent %

Reply:
can the existing `efficiency_percentage_sensor` be used instead?